templates/help.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% load static %}
+{% block extra_css %}
+{% endblock extra_css %}
```

Comment:
As aforementioned by Jason, if a block is empty, don't bother adding the block tags.